Alignment/CommonAlignmentProducer/plugins/GlobalTrackerMuonAlignment.cc

```diff
@@ -108,108 +114,101 @@ using namespace reco;
-  void debugTrajectorySOSv(const std::string, TrajectoryStateOnSurface);
-  void debugTrajectory(const std::string, Trajectory&);
-
-  void gradientGlobal(GlobalVector&, GlobalVector&, GlobalVector&, GlobalVector&, GlobalVector&, AlgebraicSymMatrix66&);
-  void gradientLocal(GlobalVector&,
-                     GlobalVector&,
-                     GlobalVector&,
-                     GlobalVector&,
-                     GlobalVector&,
-                     CLHEP::HepSymMatrix&,
-                     CLHEP::HepMatrix&,
-                     CLHEP::HepVector&,
-                     AlgebraicVector4&);
-  void gradientGlobalAlg(GlobalVector&, GlobalVector&, GlobalVector&, GlobalVector&, AlgebraicSymMatrix66&);
-  void misalignMuon(GlobalVector&, GlobalVector&, GlobalVector&, GlobalVector&, GlobalVector&, GlobalVector&);
-  void misalignMuonL(GlobalVector&,
-                     GlobalVector&,
-                     GlobalVector&,
-                     GlobalVector&,
-                     GlobalVector&,
-                     GlobalVector&,
-                     AlgebraicVector4&,
-                     TrajectoryStateOnSurface&,
-                     TrajectoryStateOnSurface&);
-  void writeGlPosRcd(CLHEP::HepVector& d3);
-  inline double CLHEP_dot(const CLHEP::HepVector& a, const CLHEP::HepVector& b) {
+
+  void gradientGlobal(
+      GlobalVector &, GlobalVector &, GlobalVector &, GlobalVector &, GlobalVector &, AlgebraicSymMatrix66 &);
+  void gradientLocal(GlobalVector &,
+                     GlobalVector &,
+                     GlobalVector &,
+                     GlobalVector &,
+                     GlobalVector &,
+                     CLHEP::HepSymMatrix &,
+                     CLHEP::HepMatrix &,
+                     CLHEP::HepVector &,
+                     AlgebraicVector4 &);
+  void gradientLocal4(GlobalVector &,
+                      GlobalVector &,
+                      GlobalVector &,
+                      GlobalVector &,
+                      GlobalVector &,
+                      CLHEP::HepSymMatrix &,
+                      CLHEP::HepMatrix &,
+                      CLHEP::HepVector &,
+                      AlgebraicVector4 &);
+  void gradientLocalGlobal();
+  void gradientGlobalAlg(GlobalVector &, GlobalVector &, GlobalVector &, GlobalVector &, AlgebraicSymMatrix66 &);
+  void writeGlPosRcd(CLHEP::HepVector &d3);
+  inline double CLHEP_dot(const CLHEP::HepVector &a, const CLHEP::HepVector &b) {
     return a(1) * b(1) + a(2) * b(2) + a(3) * b(3);
   }
+  static void fillDescriptions(edm::ConfigurationDescriptions &);
 
 private:
   void beginJob() override;
-  void analyze(const edm::Event&, const edm::EventSetup&) override;
+  void analyze(const edm::Event &, const edm::EventSetup &) override;
   void endJob() override;
 
   // ----------member data ---------------------------
-  const edm::ESGetToken<GlobalTrackingGeometry, GlobalTrackingGeometryRecord> m_TkGeometryToken;
```

Comment:
Why this change. This is undoing https://github.com/cms-sw/cmssw/commit/233abdec44ee94ce9036e6b731235593aa82e0bc